docs/integration_resources/resources/usd.md

```diff
@@ -0,0 +1,29 @@
+# USD
+
+Shift has developed a specialized catalog and utility plugins for using the USD API in Shift. 
+
+* [Shift_USD](https://github.com/Inbibo/Shift_USD)
+
+## Installation
+
+To start working with the USD API in Shift, please follow the following steps:
+
+* Download the necessary dependencies (Python & USD).
+* Clone the [Shift_USD](https://github.com/Inbibo/Shift_USD) repository into the preferred folder.
+* Add the path to the Shift_USD directory to the `PYTHONPATH` environment variable.
+* Add the path to the Shift_USD directory to the `SHIFT_CATALOG_PATH` environment variable.
+* Add the path of the *plugin.json* file found in the Shift_USD directory to the `SHIFT_PLUGIN_PATH` environment variable.

```

Comment:
Question here, using Shift inside a DCC is not the case, but trying to use Shift as standalone application, it's necessary to add the path to the `bin` and `lib` folders from compiled `usd` to the `PATH` environment variable too to make it work. Should we add that extra line?

Reply:
Yes, you're right. When they download the resources for installing USD, in the webpage should be the necessary information to install it correctly, is not on us. 👍